scripts/create-project.sh

```
@@ -1,119 +1,119 @@
-done
-
-# Reinitialize git
-cd "$destination_dir"
-rm -rf .git
-git init
-git add .
-git commit -m "Initial commit of $new_project_name"
-
-echo "Project cloned successfully!"
-echo "New project created at: $destination_dir"
+#!/bin/bash
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)